sim/dex-abilities.ts

```diff
@@ -12,7 +12,7 @@ export interface AbilityData extends Partial<Ability>, AbilityEventMethods, Even
 	name: string;
 }
 
-export type ModdedAbilityData = AbilityData | Partial<AbilityData> & {inherit: true};
+export type ModdedAbilityData = AbilityData | Partial<AbilityData> & {inherit: true, gen?: number};
```

Comment:
I'm pretty sure this is unnecessary. `AbilityData` already contains `gen`:

![image](https://user-images.githubusercontent.com/551184/102003379-9584ee80-3cbb-11eb-9bb3-484ecebe4a27.png)

Reply:
bump

Reply:
yeah @Zarel seems fine to just nuke it.

I'll leave it with @KrisXV to look at it since rn its 2am for me

Reply:
nuked.